Dockerfile

```
@@ -101,7 +124,9 @@ USER root
 RUN --mount=target=/host \
     mkdir -p /local/rpms /local/migrations ./rpmbuild/RPMS \
     && ln -s /host/build/rpms/*.rpm ./rpmbuild/RPMS \
-    && find /host/build/rpms/ -maxdepth 1 -type f -name "bottlerocket-${ARCH}-migrations-*.rpm" -not -iname '*debuginfo*' -exec cp '{}' '/local/migrations/' ';' \
+    && find /host/build/rpms/ -maxdepth 1 -type f \
+        -name "bottlerocket-${ARCH}-migrations-*.rpm" \
+        -not -iname '*debuginfo*' -exec cp '{}' '/local/migrations/' ';' \
```

```suggestion
        -not -iname '*debuginfo*' \
        -exec cp '{}' '/local/migrations/' ';' \
```
nit: the exec is critically different than the find criteria.